### PR TITLE
CLDR-16936 For personNameTest files: Add copyright headers, _readme, _header

### PR DESCRIPTION
**Pull request by pedberg-icu**
CLDR-16936

- [x] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

For files in common/testData/personNameTest/
- Add copyright headers
- Add a _readme.txt (as for other directories in common/testData/)
- Add a _header.txt that can be added to additional files generated when importing to ICU (e.g. category.txt)

### CODE REVIEW

**Comment by pedberg-icu** [comment]
> There needs to be a follow-up that changes the generation program

Filed https://unicode-org.atlassian.net/browse/CLDR-16938 for that

File: common/testData/personNameTest/_header.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Test data for unit conversions
```
Comment (richgillam) [inline common/testData/personNameTest/_header.txt:1]: Unit conversions?

File: common/testData/personNameTest/af.txt [hunk 2]
```diff
@@ -1,3 +1,9 @@
+# Test data for unit conversions
```
Comment (richgillam) [inline common/testData/personNameTest/af.txt:1]: Uh-oh, looks like we have "unit conversions" instead of "personName formats" on most of the files...

**Review by richgillam — COMMENTED** [review]
Looks good, but I see one small problem and have a couple of other thoughts...
- Most of the files have a line at the top of the copyright notice that says "Test data for unit conversions".
- I was going to mention that the tools that generate these files are going to need to be modified to emit this copyright notice, but Mark beat me to that and you filed a new Radar-- maybe that's how we fix the "unit conversions" thing.
- The test on the ICU side will probably have to be fixed to recognize and skip the comment lines here.  I'm planning to do a bunch of other personNames fixes today; I'll roll that fix in with them.
- I assume we also need the copyright notice in the "catalog.txt" file that's generated by the ant file in "tools".  I also try to figure how to do that as part of my personNames fixes today.

**Comment by macchiati** [comment]
Yes, I can do that in the tool changes. I'll add a note.

File: common/testData/personNameTest/_header.txt (hunk 1 above)
Comment (pedberg-icu) [reply common/testData/personNameTest/_header.txt:1]: Yeah, I just noticed that. I will make another PR to fix, sorry about that. But at least I can check the updated process that adds the header for catalog.txt etc. and verify that the copyright scan now passes in ICU.

File: common/testData/personNameTest/af.txt (hunk 2 above)
Comment (pedberg-icu) [reply common/testData/personNameTest/af.txt:1]: Yep, will fix.

**Comment by pedberg-icu** [comment]
> Looks good, but I see one small problem and have a couple of other thoughts...

@richgillam Thanks.
* Yes, I noticed the incorrect line about unit tests, will fix in another PR for this ticket.
* I did change the code that generates catalog.txt to add the header, and changed ExhaustivePersonNameFormatterTest to skip those lines in the catalog file. See the latest iteration of https://github.com/unicode-org/icu/pull/2530/

**Comment by richgillam** [comment]
> * Yes, I noticed the incorrect line about unit tests, will fix in another PR for this ticket.
Thanks a lot!

> * I did change the code that generates catalog.txt to add the header, and changed ExhaustivePersonNameFormatterTest to skip those lines in the catalog file. See the latest iteration of [ICU-22325 Integrate CLDR release-44-m1 to ICU icu#2530](https://github.com/unicode-org/icu/pull/2530)

Didn't realize you'd rolled all that stuff into the big CLDR-to-ICU PR.  Just reviewed it and it looks good.  Thanks again (and apologies again!) for the work on the PersonName stuff!  I'm going to try to spend the rest of today bringing the PersonName stuff back up to date (and maybe regenerating the exhaustive-test data files, if I can figure out how to do that), and I'm hoping that'll enable me to un-comment all the test cases you had to comment out.

**Review by srl295 — COMMENTED** [review]
PMLGTM other than naming fix

**Pull request merged**